package.json

```diff
@@ -94,11 +94,14 @@
     "webpack": "^1.13.2"
   },
   "dependencies": {
+    "aphrodite": "^0.5.0",
```

Comment:
Doesn't this mean that everyone has to install aphrodite even when not using it?